GitUI/UserControls/RevisionGrid.cs

```diff
@@ -1693,12 +1693,12 @@ public struct DrawRefArgs
                         int gravatarTop = e.CellBounds.Top + textHeight + 6;
                         int gravatarLeft = e.CellBounds.Left + baseOffset + 2;
 
-                        var gravatar = _avatarCache.GetImage($"{revision.AuthorEmail}.png", null);
+                        var gravatar = _gravatarService.GetImage(revision.AuthorEmail, null);
```

Comment:
Yes, I'm still in a bit of conundrum about this...
The service knows how to translate the email into the image name, and the cache deals with images by name. Though it doesn't feel right to expose `GetImageName(email)` method at the service level...
I think I will have extract that functionality into a separate class to have it reusable